Error reporting (#50)
piiano/flows: cli/README.md

1. `FLOWS_SKIP_ENGINE` - Setting to `true` skips the engine and runs just the viewer on the existing report (This variable is only used by the flows-cli script).
2. `PIIANO_CS_VIEWER_MODE` -  
      Setting to `none` skips the viewer causing the script to exit after scanning (This variable is only used by the flows-cli script).  
      Setting to `local` (default) opens the flows viewer locally.  
      Setting to `online` opens the report in piiano cloud (coming soon).

> 3. `SCAN_NAME` - if `PIIANO_CS_VIEWER_MODE` = `online` , sets the scan name in the cloud.

Reviewer: ```suggestion
3. `SCAN_NAME` - if `PIIANO_CS_VIEWER_MODE` = `online` , sets the scan name for the online viewer.
```
